package.json

```diff
@@ -101,25 +102,16 @@
   "lint-staged": {
     "*.js": [
       "eslint",
-      "prettier --write",
-      "git add"
+      "prettier --write"
     ],
     "*.css": [
       "stylelint",
-      "prettier --write",
-      "git add"
+      "prettier --write"
```

Comment:
Can you delete prettier for css please? I have a fix for this on v7, prettier lint breaks commits with css modified.

Reply:
Done